modules/videoio/test/test_audio.cpp

```diff
@@ -280,7 +284,8 @@ const paramCombination mediaParams[] =
 };
 
 INSTANTIATE_TEST_CASE_P(/**/, Media, testing::ValuesIn(mediaParams));
-#endif  // _WIN32
+
+#ifdef _WIN32
```

Comment:
Prefer backend check if possible.

(because GStreamer builds exist for Windows too)

Reply:
_WIN32 check is only for MSMF

Reply:
As I wrote above, GStreamer **for Windows** passes this check too.

> is only for MSMF

it is not.